start.bat

```diff
@@ -1,4 +1,4 @@
-@echo off
+rem @echo off
```

Comment:
jrd have headless mode. Maybe enable echo on when  --verbose is set?

Reply:
That might have been a forgotten comment from debugging. But you'll have to tell me more about the mode.